web/server.js

```diff
@@ -1,3 +1,5 @@
-import { start } from '@/web/MoveServer';
+import config from '@/lib/config/MoveConfig';
```

Comment:
can we rename this file to `web.js`, following the convention elsewhere?

`server.js` is no longer as useful of a name as it was when this was first created, and it's about to become an even less useful name once we introduce `scheduler`.